test/extended/templates/templateinstance_security.go

```diff
@@ -67,180 +68,216 @@ var _ = g.Describe("[Conformance][templates] templateinstance security tests", f
 		}
 	)
 
-	g.BeforeEach(func() {
-		adminuser = createUser(cli, "adminuser", bootstrappolicy.AdminRoleName)
-		edituser = createUser(cli, "edituser", bootstrappolicy.EditRoleName)
-		editgroup = createGroup(cli, "editgroup", bootstrappolicy.EditRoleName)
-		editbygroupuser = createUser(cli, "editbygroupuser", "")
-		addUserToGroup(cli, editbygroupuser.Name, editgroup.Name)
-	})
+	g.Context("", func() {
```

Comment:
Not sure what an empty string here means.

Reply:
it's an idiom that devex is currently using in tests to ensure that the inner AfterEach runs before k8s' global AfterEach destroys the namespace

Reply:
Sounds crazy.  But I guess that is ginko...